audit/event.go

```diff
@@ -109,3 +141,15 @@ func (st subtype) MetricTag() string {
 
 	return ""
 }
+
+// HumanReadableString returns the subtype as a human-readable string.
+func (t subtype) HumanReadableString() string {
```

Comment:
This could just be named String(), no?  It feels like human-readable is the spirit behind the Stringer interface.

Reply:
🤦🏼